protocols/rendezvous/src/codec.rs

```diff
@@ -254,6 +254,7 @@ impl Decoder for RendezvousCodec {
     }
 }
 
+#[allow(clippy::enum_variant_names)]
```

Comment:
I wouldn't mind breaking this API and fix the lint instead. We make new minor releases of `libp2p-swarm` all the time.

Just add a changelog entry please if you decide to do it :)

Reply:
Great, will do! :)